### PR TITLE
add NER model based on BERT

### PR DESCRIPTION
**Pull request by rickycao-qy**

### CODE REVIEW

File: packages/plugins/model-evaluate/tensorflow-bert-ner-model-evaluate/package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  "devDependencies": {
+    "@types/jasmine": "^3.5.7",
+    "nyc": "14.1.1"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "pipcook": {
+    "category": "modelTrain",
+    "datatype": "image"
```
Comment (WenheLI) [inline packages/plugins/model-evaluate/tensorflow-bert-ner-model-evaluate/package.json:29]: This should be text?
```suggestion
    "datatype": "text"
```

File: packages/plugins/model-train/tensorflow-bert-ner-model-train/package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  "devDependencies": {
+    "@types/jasmine": "^3.5.7",
+    "nyc": "14.1.1"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "pipcook": {
+    "category": "modelTrain",
+    "datatype": "image"
```
Comment (WenheLI) [inline packages/plugins/model-train/tensorflow-bert-ner-model-train/package.json:29]: Ditto
```suggestion
    "datatype": "text"
```

File: packages/plugins/model-define/tensorflow-bert-ner-model-define/package.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  "devDependencies": {
+    "@types/jasmine": "^3.5.7",
+    "nyc": "14.1.1"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "pipcook": {
+    "category": "modelDefine",
+    "datatype": "image"
```
Comment (WenheLI) [inline packages/plugins/model-define/tensorflow-bert-ner-model-define/package.json:30]: Ditto
```suggestion
    "datatype": "text"
```

**Comment by rickycao-qy** [comment]
@WenheLI Fix this

File: packages/core/src/utils/public.ts [hunk 4]
```diff
@@ -116,6 +116,7 @@ export async function downloadAndExtractTo(resUrl: string): Promise<string> {
 
   const { protocol, pathname } = url.parse(resUrl);
   const destPath = path.join(PIPCOOK_TMPDIR, generate());
+  await fs.ensureDirSync(destPath);
```
Comment (yorkie) [inline packages/core/src/utils/public.ts:119]:
```suggestion
  await fs.ensureDir(destPath);
```

Comment (rickycao-qy) [reply packages/core/src/utils/public.ts:119]: fixed

**Comment by rickycao-qy** [comment]
Has moved into seperate repo

**Comment by yorkie** [comment]
> Has moved into seperate repo

What's the repo URL?

**Comment by rickycao-qy** [comment]
https://github.com/imgcook/pipcook-plugin-tensorflow-bert-ner-model

**Comment by rickycao-qy** [comment]
https://github.com/imgcook/pipcook-plugin-tensorflow-bert-ner-model
https://github.com/imgcook/pipcook-plugin-tensorflow-bert-ner-model-train
https://github.com/imgcook/pipcook-plugin-tensorflow-bert-ner-model-evaluate

**Pull request closed without merging**